src/main/resources/application-heroku.properties

```diff
@@ -0,0 +1,3 @@
+spring.datasource.url=jdbc:mysql://us-cdbr-east-06.cleardb.net/heroku_da4d74ab6e6d198?reconnect=true
+spring.datasource.username=ba6fb66ee506c2
+spring.datasource.password=4c760aba
```

Comment:
These really shouldn't be checked in, can we pull them from an environment variable?

This may be a helpful guide
https://devcenter.heroku.com/articles/preparing-a-spring-boot-app-for-production-on-heroku#remove-secrets-from-source-code